addons/portal/views/portal_templates.xml

```diff
@@ -10,7 +10,7 @@
                 <nav class="navbar navbar-expand navbar-light bg-light">
                     <div class="container">
                         <a href="/" class="navbar-brand logo">
-                            <img t-att-src="'/logo.png?company=%s' % res_company.id" t-att-alt="'Logo of %s' % res_company.name" t-att-title="res_company.name"/>
+                            <img t-if="not res_company.uses_default_logo" t-att-src="'/logo.png?company=%s' % res_company.id" t-att-alt="'Logo of %s' % res_company.name" t-att-title="res_company.name"/>
```

Comment:
shouldn't we move the condition on the `<a>` element ?

Reply:
Oh actually this shouldn't be conditional at all ^^'

I'll go over these again to make sure I only touch emails